package.json

```diff
@@ -1,57 +1,65 @@
 {
-  "name":        "crystal-ameba",
+  "name": "crystal-ameba",
   "displayName": "crystal-ameba",
-  "publisher":   "veelenga",
+  "publisher": "veelenga",
   "description": "Ameba extension for Visual Studio Code",
-  "version":     "0.0.5",
-  "license":     "MIT",
-  "engines":     {
+  "version": "0.0.5",
+  "license": "MIT",
+  "engines": {
     "vscode": "^1.22.0"
   },
   "repository": {
     "type": "git",
-    "url":  "https://github.com/crystal-ameba/vscode-crystal-ameba.git"
+    "url": "https://github.com/crystal-ameba/vscode-crystal-ameba.git"
   },
   "categories": [
     "Programming Languages",
     "Linters"
   ],
   "activationEvents": [
     "onLanguage:crystal",
-    "onCommand:crystal.ameba"
+    "onCommand:crystal.ameba.lint"
   ],
-  "main":        "./out/extension.js",
+  "main": "./out/extension.js",
   "contributes": {
     "languages": [
       {
-        "id":         "crystal",
+        "id": "crystal",
         "extensions": [
           ".cr"
         ]
       }
     ],
     "commands": [
       {
-        "command": "crystal.ameba",
-        "title":   "Crystal: lint by Ameba"
+        "command": "crystal.ameba.lint",
+        "title": "Crystal Ameba: run lints"
+      },
+      {
+        "command": "crystal.ameba.restart",
+        "title": "Crystal Ameba: restart extension"
+      },
+      {
+        "command": "crystal.ameba.disable",
+        "title": "Crystal Ameba: disable lints (workspace)"
```

Comment:
what do you think about disabling the extension automatically when the ameba executable is not found?

Reply:
I think having the button option in the error message should be fine, especially if you're just installing Ameba in your project which shouldn't take long.